ARImageTrackingExamples/ARImageTrackingExamples/Helpers/UIViewController+helpers.swift

```diff
@@ -9,8 +9,15 @@ extension UIViewController {
 
 extension Float {
     static let smallOffset: Float = 0.01
+    static let rightAngle: Float = .pi / 2
 }
 
 extension CGFloat {
     static let slightyTransparent: CGFloat = 0.95
+    static let half: CGFloat = 0.5
+    static let third: CGFloat = 0.75
+}
+
+extension Int {
+    static let renderFirst: Int = -1
```

Comment:
this extension might live better nearer where it's used. Whilst something like `Float.half` has context everywhere, this `Int.renderFirst` seems to only to make sense in AR, so it should probably be private in the file it is used